doc-content/enterprise-only/bpmn/bpmn-subprocesses-ref.adoc

```diff
@@ -1,87 +1,54 @@
 
-
-* Attributes
-+
-*AdHocCompletionCondition*
-+
-When this condition evaluates to `true`, the execution finishes.
-+
-*AdHocOrdering*
-+
-Enables you to choose paralel or sequential execution of elements inside of the sub-process. 
-+
-*Variable Definitions*
-+
-Enables you to define process variables available only for elements of the sub-process.
+[[_event_sub_process]]
+.Event subprocess
 
 
-[[_multiple_instances]]
-.Multi-instance Sub-Process
-
-A Multiple Instances Sub-Process is instantiated multiple times when its execution is triggered. The instances are created in a sequential manner, that means a new sub-process instance is created only after the previous instance has finished.
-
-A Multiple Instances Sub-Process has one incoming connection and one outgoing connection.
-
-
-* Attributes
-+
-*MI collection input*
-A collection to be iterated through. It is used to create individual instances of given activity. The sub-process will be run with each element of this collection.
-+
-*MI collection output*
-A collection of the sub-process execution results.
-+
-*MI completion condition*
-An MVEL expression evaluated at the end of every instance. When evaluated as `true`, the sub-process is evaluated as finished and the sub-process's outgoing flow is taken. Possible remaining sub-process instances are cancelled.
-+
-*MI data input*
-A variable name for each element from the collection that will be used in the process. 
-+
-*MI data output*
-An optional variable name for the collection of the results. 
+An event subprocess becomes active when its start event is triggered. It can interrupt the parent process context or run in parallel with it.
 
-[[_event_sub_process]]
-.Event Sub-Process
+With no outgoing or incoming connections, only an event or a timer can trigger the subprocess. The subprocess is not part of the regular control flow.
+Although self-contained, it is executed in the context of the bounding process.
 
+Use an event subprocess within a process flow to handle events that happen outside of the main process flow.
+For example, while booking a flight, two events may occur:
 
-An event sub-process becomes active when its start event gets triggered. It can interrupt the parent process context or run in parallel to it.
+* Cancel booking (interrupting)
+* Check booking status (non-interrupting)
 
-With no outgoing or incoming connections, only an event or a timer can trigger the sub-process. The sub-process is not part of the regular control flow.
-Although self-contained, it is executed in the context of the bounding sub-process.
+You can model both of these events using the event subprocess.
 
-Use an event sub-process within a process flow to handle events that happen outside of the main process flow.
-For example, while booking a flight, two events may occur:
+[[_multiple_instances]]
+.Multiple instance subprocess
```

Comment:
"Multiple-instance subprocess" Search all. You switch between multiple instances subprocess, multiple instance subprocess, multiple-instance subprocess.

Also, something funky in this sentence: "in a sequentially (a new subprocess ... )." Something must have fallen out.